packages/react-email/src/utils/static-node-modules-for-vm.ts

```diff
@@ -71,6 +72,7 @@ export const staticNodeModulesForVM = {
   'node:http': http,
   fs,
   'node:fs': fs,
+  'fs/promises': fs.promises,
```

Comment:
Could you just add in the `node:` prefixed variant here?

Reply:
Done: 9a559e85d72d4c4de835a272460a0de232182858